package.json

```
@@ -1,22 +1,31 @@
 {
-  "name": "msal-react-quickstart",
+  "name": "ms-identity-javascript-react-spa",
   "version": "0.1.0",
-  "description": "A React single-page application calling Microsoft Graph API using MSAL.js (w/ AAD v2 endpoint)",
   "private": true,
   "dependencies": {
-    "@azure/msal-browser": "^2.15.0",
-    "@azure/msal-react": "^1.0.1",
-    "bootstrap": "^4.5.3",
-    "jquery": "^3.5.1",
-    "popper.js": "^1.16.1",
-    "react": "^17.0.1",
-    "react-bootstrap": "^1.3.0",
-    "react-dom": "^17.0.0",
-    "react-scripts": "^4.0.2"
+    "@azure/msal-browser": "^2.28.3",
+    "@azure/msal-react": "^1.4.7",
+    "@testing-library/jest-dom": "^5.16.5",
+    "@testing-library/react": "^13.4.0",
+    "@testing-library/user-event": "^13.5.0",
```

```suggestion
    "@testing-library/jest-dom": "^5.16.5",
    "@testing-library/react": "^13.4.0",
    "@testing-library/user-event": "^13.5.0",
```
Don't think we need testing library deps in this one